.travis.yml

```diff
@@ -117,3 +122,17 @@ jobs:
       before_script: skip
       script:
         - beaver bintray upload -D $DIST_VERSION -N $CACHE_DIR/*.deb
+    - <<: *deploy
+      # Upload code coverage report to codecov.io for upstream major, minor
+      # branches and git tags
+      stage: deploy
+      if: tag IS present
+          OR branch =~ ^v[0-9]+\.(x|[0-9]+)\.x$
+          AND NOT type IN (pull_request)
+          AND repo = sociomantic-tsunami/libdrizzle-redux
+      env:
+          MAKE_TARGET="$MAKE_TARGET:code-coverage-capture" CONFIGURE_ARGS=--enable-code-coverage
+      before_script: skip
+      script:
+        - beaver docker build -f "docker/Dockerfile.ubuntu.xenial" .
+        - BEAVER_CODECOV_REPORTS="$CACHE_DIR/libdrizzle-redux-coverage.info" beaver codecov -X gcov -Z
```

Comment:
I think this is wrong. Why are you just publishing reports for this repo and just if it's not a PR. Coverage should be published for every matrix build ideally, codecov will merge the reports. For different builds you can even [use the `-F` flag](https://github.com/codecov/codecov-bash/blob/8b76995ad4a95a61cecd4b049a448a402d91d197/codecov#L129-L133) so you can group which build cover which parts.

See for example: https://github.com/sociomantic-tsunami/ocean/blob/be9450ef600863d0c9a02c69f888872ddf01ac1e/.travis.yml#L38